apps/configurator/priv/repo/migrations/20250414200744_add_level_up_configuration.exs

```diff
@@ -0,0 +1,6 @@
+defmodule Configurator.Repo.Migrations.AddLevelUpConfiguration do
+  use Ecto.Migration
+
+  def change do
+  end
+end
```

Comment:
What's this ghost migration?

Reply:
I think it got created with `mix ecto.gen migration` and I added it by mistake